src/crawler.ts

```diff
@@ -1556,22 +1556,43 @@ self.__bx_behaviors.selectMainBehavior();
         throw new Error("page response missing");
       }
 
+      const respUrl = resp.url();
+      const isChromeError = page.url().startsWith("chrome-error://");
+
+      if (respUrl !== url && depth === 0 && !isChromeError) {
```

Comment:
Very minor suggestion, I would rewrite it `if (depth === 0 && !isChromeError && respUrl !== url ) {` for clarity

Reply:
```suggestion
      if (depth === 0 && !isChromeError && respUrl !== url) {
```